torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -1843,8 +1843,9 @@ void InitXlaModuleBindings(py::module m) {
                                   py::call_guard<py::gil_scoped_release>());
   m.def("_xla_get_distributed_runtime_service",
         [](int num_nodes) -> std::unique_ptr<xla::DistributedRuntimeService> {
+          std::string port = runtime::sys_util::GetEnvString("COORDINATOR_PORT", "8547");
           std::string dist_service_addr =
-              runtime::sys_util::GetEnvString("PJRT_DIST_SERVICE_ADDR", "");
```

Comment:
How about we make the address and port parameters to `_xla_get_distributed_runtime_service`? I'll need this shortly for autocheckpointing, so happy to pick up the refactor after you land this PR.

Reply:
per our discussion offline, let's push it down to the cpp layer

Reply:
Do you plan to take this in a follow-up PR? If so, I can drive it to unblock autocheckpointing, the GPU-related changes to the distributed runtime seem complete (feel free to let me know otherwise).

Reply:
Yes, it's in https://github.com/pytorch/xla/pull/5714.